saleor/account/forms.py

```diff
@@ -9,6 +10,16 @@
 from .i18n import AddressMetaForm, get_address_form_class
 
 
+class FormWithReCaptcha(forms.BaseForm):
+    def __new__(cls, *args, **kwargs):
+        if settings.ENABLE_RECAPTCHA:
```

Comment:
Can we get rid of `ENABLE_RECAPTCHA` bool in favor of checking the presence of private and public recaptcha keys?

Reply:
Well... Yes, we could set the keys to None. And then check if both are set. Thus, users will no longer have access to the default development keys. 
If we do like so, we need to update the docs with the developments keys as note.

As you prefer.